telemetry/service/service-model.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
I'm excited for this, we can do vscode and jb next to no longer duplicate this file :D